ipv-api/src/main/java/uk/gov/di/authentication/ipv/lambda/IPVCallbackHandler.java

```diff
@@ -309,6 +310,34 @@ public APIGatewayProxyResponseEvent handleRequest(
 
             var userIdentityError = validateUserIdentityResponse(userIdentityUserInfo);
             if (userIdentityError.isPresent()) {
+                var accountInterventionStatus =
+                        getAccountInterventionStatus(pairwiseSubject.getValue());
+                if (configurationService.isAccountInterventionServiceEnabled()) {
+                    if (accountInterventionStatus.blocked()) {
+                        // TODO: back channel logout + redirect to blocked page
+                        LOG.info("Account is blocked");
+                    } else if (accountInterventionStatus.suspended()
+                            || accountInterventionStatus.resetPassword()
+                            || accountInterventionStatus.reproveIdentity()) {
+                        // TODO: back channel logout + redirect to suspended page
+                        LOG.info(
```

Comment:
There are two cases which can generate the same log messages: when `errorObject.isPresent()` and `userIdentityError.isPresent()`. Would it be better to make these messages unique to help debugging?

Reply:
They are both in relation to the AIS result so when this functionality is extracted out both locations should be calling the same function -> will be using the same logging message -> should be clear that it is from the AIS result :)

Reply:
They also produce different logs and the `errorObject.isPresent()` situation additionally audits the error event in the `generateAuthenticationErrorResponse` method